README.md

```diff
@@ -153,6 +153,13 @@ $ swarmd -d /tmp/node-2 --hostname node-2 --join-addr 127.0.0.1:4242 --join-toke
 $ swarmd -d /tmp/node-3 --hostname node-3 --join-addr 127.0.0.1:4242 --join-token <Worker Token>
 ```
 
+If joining as a manager (and on the same node), specify the listen-control-api as the one specified
+for the first node (along with the listen-remote-api with a different port).
+
+```sh
+$ swarmd -d /tmp/node-4 --hostname node-4 --join-addr 127.0.0.1:4242 --join-token <Manager Token> --listen-control-api /tmp/node-1/swarm.sock --listen-remote-api 127.0.0.1:4245
```

Comment:
Shouldn't this be `/tmp/node-4/swarm.sock`? If it works for multiple nodes to share the same control socket, it's accidental. Each should have its own.